apps/allocations/app/components/Card/Account.js

```diff
@@ -14,6 +14,8 @@ import {
   theme,
 } from '@aragon/ui'
 
+const ETH_DECIMALS = BigNumber(10e17)
```

Comment:
Better reuse common constants into maybe utils/constant file instead of having the same value declared over an over on several files